src/registrar/views/domain_request.py

```diff
@@ -355,7 +369,7 @@ def db_check_for_unlocking_steps(self):
                 or self.domain_request.zipcode is not None
                 or self.domain_request.urbanization is not None
             ),
-            "about_your_organization": self.domain_request.about_your_organization is not None,
+            "about_your_organization": True,
```

Comment:
Why set this to always be true? Doing so results in the "About your Organization" immediately populating with a check on the sidebar (after selecting "interstate" for generic org type) even though it hasn't been visited. Is that the behavior we want?